docs/self-hosting.md

```diff
@@ -11,7 +11,7 @@ platform.
 
 We aren't quite ready for mainstream adoption of our platform,
 [Forem](https://forem.com), but that hasn't stopped some adventurous developers
-from standing up new communities.
+from starting up new communities.
```

Comment:
this was actually correct, "to stand up" means "to erect"

Reply:
Yeah OK, I was wondering if I was just unfamiliar with it! Thanks for clarifying.